scripts/cpu-stats.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+#
+# Reports cpu usage statistics
+#
+set -e
+
+[[ $(uname) == Linux ]] || exit 0
+
+cd "$(dirname "$0")"
+
+# shellcheck source=scripts/configure-metrics.sh
```

Comment:
nit: if you `cd "$(dirname "$0")/.."` instead then this shellcheck goes away

Reply:
This doesn't working. `net-stat.sh` has the same shellcheck silence

Reply:
Fixed it! :)